governance/roles.md

```diff
@@ -0,0 +1,96 @@
+# Roles within the TAG App Delivery
+
+The TAG includes several key roles that are critical to the group's success.
+Beyond the roles described in this document the TAG has many contributors, all serving in varying capacities.
+
+- [Roles within the TAG App Delivery](#roles-within-the-tag-app-delivery)
+    - [Chair](#chair)
+    - [Technical lead](#technical-lead)
+    - [Working Group Lead](#working-group-lead)
+    - [TOC Liaison](#toc-liaison)
+    - [Emeritus Chair](#emeritus-chair)
+
+## Chair
+
+The (normally three) Co-Chairs of the TAG App Delivery drive the CNCF community efforts of the TAGs domain.
+Chairs are experienced leaders within the CNCF community with rooted expertise in the TAG area. Chairs serve a two-year term but may renew their term by submitting a follow-up application.
+Chair applications go through an [election process](https://github.com/cncf/toc/blob/master/tags/cncf-tags.md#elections) with a final CNCF TOC vote. The election process is defined [here](./chair-election-process.md)(WIP).
+
+Co-Chairs are expected to:
+* Manage the operations and governance of the group.
+* Organize, host, plan and facilitate TAG meetings and events.
+* Reporting to the CNCF TOC on the status of TAG work.
+* Encouraging community members to start projects and working groups.
+* Resolve technical difficulties and decisions related to multiple sub-projects.
+* Remind the community of the scope of the TAG and point out guardrails in discussions, working groups, and projects.
+* Serve as TAG leadership representative in CNCF project discussions and at community events.
+* Evolve the TAG to reflect ongoing changes in the industry.
+* Onboard and mentor new community members.
+* Mentor community members in a leadership role within the TAG.
+* Enforce and promote diversity in TAG work.
+* Enforce and promote good communication in TAG efforts in accordance with the [CNCF CoC](https://www.cncf.io/conduct/)
+
+Permissions:
+* Chairs have admin access to the TAG GitHub repository.
+
+## Technical Lead
+
+A technical lead (TL) expands the bandwidth of the leadership team in terms of covering technical areas, growing the community, organizing events or doing TAG communications.
+
+Proposals must have Chair as an active sponsor and the agreement of the majority of the current leadership team (Chairs, Tech Leads & Liaisons). TLs are experienced contributors within the CNCF community with rooted expertise in the TAG area. 
```

Comment:
Can we define the experience the contributors should have? See examples above for Chairs, should similar ones apply to TLs?